apps/ledger-live-mobile/src/screens/Platform/v2/Catalog/AppCard.tsx

```diff
@@ -9,6 +9,8 @@ import { AppIcon } from "../AppIcon";
 import { Theme } from "../../../../colors";
 import { BackgroundGradientHorizontal } from "~/components/TabBar/BackgroundGradient";
 import { Cta } from "./Cta";
+import { useLocale } from "~/context/Locale";
+import { translateContent } from "@ledgerhq/live-common/platform/logic";
```

Comment:
We should probably use the one from the wallet-api folder since we are in the v2 folder of platform in this file
```suggestion
import { translateContent } from "@ledgerhq/live-common/wallet-api/logic";
```